irohad/torii/processor/CMakeLists.txt

```diff
@@ -11,4 +11,5 @@ target_link_libraries(processors PUBLIC
     shared_model_proto_builders
     status_bus
     common
+    verified_proposal_creator_common
```

Comment:
Since you have been touching the file, please add license header.

Reply:
https://github.com/hyperledger/iroha/blob/fe542f16b69c0962943ed9a808dc2a31554308be/irohad/torii/processor/CMakeLists.txt#L1-L2